tests/test_interp_relperm.py

```diff
@@ -375,12 +332,11 @@ def test_wrong_family(tmp_path):
         "family": "Rockefeller",
         "delta_s": 0.1,
     }
-    with pytest.raises(
-        SystemExit, match="Is x a number is false on input 'Rockefeller'"
-    ):
+    with pytest.raises(ValidationError):
```

Comment:
Should we keep the `match="Is x a number is false on input 'Rockefeller'"` from the old code, porting the error text to what Pydantic spits out of course?